### DISCUSSION TITLE
Reference used to build database of Xray_lines intensity weights?

### DISCUSSION THREAD

**Discussion by Andrea-gm** [hyperspy/exspy#85] [Q&A] [answered]
I was wondering if somebody knows from which source the intensity weights of elemental xray lines have been taken. They can be found at exspy.material.elements.(ELEMENT SYMBOL HERE).Atomic_properties.Xray_lines

Also, somehow the M lines are missing for elements from Z = 30 (Zn) to Z = 56 (Ba)

**Comment by ericpre** [comment, marked as answer]
Based on [a comment](https://github.com/hyperspy/exspy/blob/ee31d0a46d9ccb16d0a137ba37f0e0fce3698b62/exspy/_misc/elements.py#L3-L11) in the code,, they come from https://www.nist.gov/pml/x-ray-form-factor-attenuation-and-scattering-tables.

It would be good to include this comment in the corresponding [docstring](https://hyperspy.org/exspy/reference/material.html#exspy.material.elements).

**Reply by Andrea-gm** [reply]
Thanks for the quick reply!

**Reply by Andrea-gm** [reply]
I confirm that the M lines are missing up to Z = 56. I suspect this is because the Ma1 lines do not exist for these elements. But the Mz do.
